setup.py

```diff
@@ -71,6 +71,7 @@
         "ciso8601",
         "click",
         "defusedxml",
+        "h5py",
```

Comment:
This is already covered up able as an extra dependency for "all" and "wagl", so shouldn't be necessary here.

Reply:
This was a change I previously made as part of the conda-forge deployment; without it, their pipeline failed